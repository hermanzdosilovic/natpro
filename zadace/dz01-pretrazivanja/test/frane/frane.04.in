1000
2524 3526 912 3412 997 4281 4133 645 2516 2564 4464 3721 96 4986 4256 3113 3740 1323 4050 4434 1088 2978 2473 3184 3513 3706 2060 4524 493 1505 1426 4013 4003 2046 4704 3659 3597 3632 514 2324 4359 4274 1279 2874 967 4571 3371 4164 4029 2909 1815 4446 3375 911 99 3120 508 4565 4847 2225 1108 2705 1842 53 4181 1296 4093 3258 3345 3840 810 550 855 121 448 4754 2495 67 1561 3772 2164 1828 865 4234 3803 3227 104 3394 191 348 1757 3763 4619 749 4939 3199 461 840 4095 4792 603 2688 90 1726 2515 808 4711 2939 2019 2466 1976 859 3416 4098 4191 1446 4727 1248 4951 1102 3699 4179 1046 151 3068 3422 1789 3661 1351 1669 1906 4844 4247 3656 4407 554 3866 1980 897 2664 2516 2560 3848 2441 4895 2140 3367 3699 2515 2725 241 2963 3920 3139 1232 201 713 3010 1027 2604 4946 3423 4194 940 4632 3709 4796 868 3668 2810 58 2853 3599 2600 2171 3604 578 4999 3084 1314 1163 2228 1743 2172 4966 4664 3668 3864 728 2692 3257 4676 3591 323 2566 4157 1466 2394 1609 4382 3235 3 3115 1577 3829 1504 1594 1485 4410 1169 1739 4505 2956 2972 2009 3485 1146 2948 1139 2731 4449 1529 3719 2691 2224 4397 1585 798 847 976 1725 3566 4438 2799 379 2403 2913 2455 793 1780 541 472 3310 4789 2817 252 1925 1946 2380 2593 3478 4463 3391 2973 4916 4144 3602 439 1464 251 2772 2690 2471 741 4363 195 2346 2979 3490 2906 2146 828 2351 3656 4386 1220 820 4961 4298 2577 429 4172 2417 930 964 3915 4753 1712 4251 1097 3433 4244 3735 1174 4718 3993 2690 2612 1888 1839 2198 27 3155 1674 2387 3350 1839 1179 2247 2827 4825 1546 570 4744 4488 656 4124 424 2288 2382 705 2703 4009 4904 2793 281 1036 2545 2317 4340 2984 3974 4797 2200 4102 3759 426 1200 4744 199 4676 1765 1904 2740 2232 3192 4290 4783 784 855 2127 626 2183 4736 1401 2997 3464 3283 1821 277 4607 4842 1431 926 3175 65 970 2857 4344 1388 3245 4404 3745 2463 2675 3510 4353 3525 1663 4586 638 4202 1784 3559 4045 1100 3377 3079 826 632 1794 4550 2957 654 1362 2896 888 860 1414 805 3763 1848 706 3440 1164 2543 2881 4490 2766 221 4162 4044 346 4793 1728 1075 1889 2413 4718 2 4967 1654 1755 1425 1153 718 842 4296 4485 3422 3539 3627 277 1926 3618 2649 3336 1474 4060 820 1078 572 881 4216 4357 1924 1827 77 446 4045 2061 1876 2546 1998 1040 3717 859 2002 4252 1726 3533 828 2836 3609 1599 2133 4501 116 1904 2619 3484 341 3628 3850 1966 2471 4265 2898 1979 4343 4019 2713 906 4754 1381 2528 165 1228 2938 673 4503 3238 4727 4770 4562 2654 3338 4109 2713 422 1077 3696 781 2036 922 2122 2848 419 4051 2731 4853 4180 913 4416 4060 3811 3576 4344 3970 1218 1230 2232 2935 3702 460 1559 640 4661 4826 4437 3292 2857 1844 216 1080 653 4383 1972 2161 1321 652 2608 1170 1028 4501 94 2465 2860 4792 660 1462 3809 3149 1646 4566 2520 923 1129 913 502 2016 1978 3069 1695 3457 3722 1664 4676 1879 3518 3777 1157 3928 1795 2905 4101 1563 1276 2680 4885 2132 2795 3800 622 4864 641 2755 1395 796 2193 166 2695 3000 4159 4819 3390 3124 2478 34 3710 2960 1257 1199 2009 3162 2701 3580 1980 4020 4719 2793 2852 2303 3183 552 2567 4078 4431 4487 2462 3067 867 2665 3934 1124 1668 2966 2905 1394 2762 1706 4688 3028 3976 1772 3778 1569 3987 2701 16 2932 2637 3836 1356 4614 1339 2196 975 757 217 2215 1542 2251 1155 4289 3720 2455 3446 3768 941 2880 3942 4970 309 3424 4206 742 289 2109 1488 1764 4538 246 4996 730 3676 4642 3447 3380 1105 3116 695 3969 517 91 2390 539 3243 4970 4176 184 2486 3963 575 1919 3979 428 1523 1990 3745 1590 537 4560 3541 671 1175 49 2263 2619 1758 641 3261 4088 4228 347 524 4709 4638 416 273 1137 1634 3293 4340 761 515 4946 1586 2311 4204 2557 1294 4822 3358 1423 3923 1706 425 3400 1200 1100 508 2055 3062 1035 4419 4670 3347 3610 2570 813 1453 1263 2403 889 3054 1109 348 3551 1981 3283 3894 4093 857 3700 602 4613 1735 2402 3578 1580 4995 99 331 3769 2506 2975 3002 2290 3666 4894 1301 2178 2322 4704 4325 3215 2796 1858 1505 103 1221 3807 528 2006 428 3945 2454 2884 3458 4757 701 13 1761 1327 4269 2176 4995 638 2905 70 1312 2946 288 4247 707 2960 490 4864 3895 4924 4297 2741 3347 3495 412 120 3064 1130 4023 1822 3483 148 1140 328 2937 1461 3353 990 3994 1681 3711 2202 1812 3269 2560 411 3348 209 3343 1802 1614 4731 4764 1654 2870 4888 284 1714 3092 2356 1954 733 4648 267 2566 949 1531 2571 2836 3781 3976 2347 2882 1242 1994 2639 2615 3836 3999 2068 2433 4237 3542 1399 2472 3883 900 3108 1155 1327 4554 3392 3220 4842 3036 25 759 2940 3931 2280 110 2480 3789 1930 2526 872 1214 4314 943 761 2944 2212 1443 1914 2228 3168 538 3031 1963 1162 780 3518 2599 37 3159 1448 3821 2535 2971 2660 3230 915 4500 1923 1902 2119 2973 626 569 3055 4749 2578 2896 725 1939 2566 741 3338 2801 2113 139 535 3035 3660 4823 347 2613 1828 2667 1760 110 3522 4387 2001 4322 10 4857 3172 4826 2961 4006 3603 2334 763 377 1939 4920 209 2576 4148 4840 4842 3927 2106 1493 4242 622 143 2079 4946 2238 4987 780 1895 3249 2695 369 4044 2844
