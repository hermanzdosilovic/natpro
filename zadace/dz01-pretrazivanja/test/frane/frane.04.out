2043252031.100
